documentation/govuk-paas.md

```diff
@@ -168,9 +168,19 @@ For all the environment variables defined in `.env.example` set them up with:
 cf7 set-env teaching-vacancies-dev ENV_VAR_NAME env_var_value
 ```
 
-When you are done setting up environment variables remember to restart the app:
+You will be asked to stage the changes. Do so with:
```

Comment:
Actually, it does not ask to `stage` the changes, it asks to `restage` the app. You can either `restart` or `restage`. A `restart` is shorter and is sufficient in most of the occasions